### PR TITLE
Speculating

### PR DESCRIPTION
**Pull request by MrAmmar**
Add new page , my own mark to the story :)

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2692)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: arabic/learn/learn.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 اذا قررت ان تكمل الالتعلم فانه عليك ان تبحث عن موضوع مكمل لموضوع السياق الماضي، هل تعرف لماذا؟
 لانه يجب عليك الاستفاده من السياق الماضي وتطوير قدراتك فيه لذا ماذا تتوقع ان تفعل الان؟
 قال صلى الله عليه وسلم من سلك طريقاً يلتمس فيه علماً سهل الله له طريقاً الى الجنة
+[و لماذا التعلم ](learn\whylearning\speculate.md)
```
Comment (casey-collab) [inline arabic/learn/learn.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: arabic/learn/whylearning/speculate.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+كم من الوقت سنتعلم ؟ أسنتوقف بعد حصولنا مبتغانا؟
```
Comment (casey-collab) [inline arabic/learn/whylearning/speculate.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by MrAmmar** [comment]
Sorry it was my bad , i used \ instead of /

**Comment by cbuckey-uda** [comment]
No problem, @MrAmmar! @casey-collab, please take another look.

**Comment by casey-collab** [comment]
Looks great! :octocat:

**Pull request merged**